### ISSUE TITLE
Is this working?

### ISSUE THREAD

**Issue by helmuth1792** [hunterjm/futgui#178]
Hey guys, I just found out about this tool, I just have a few questions.
I have been trying the tool, but I don't know how to get it to buy, I used the tutorial, but when I get it to run, it doesn't seem like it is buying anything, my coins don't move, not even when I put a really high price for a player I know is cheaper.
the other question is, can this tool be used to only snipe one card? For example I just want a cheap aguero, if I let it running will it just snipe me the cheapest one?

thanks a lot

**Comment by mestesso93** [comment]
yes, you must click on bid snipe. With this function, the program doesn't make auction, but only check with bin price.

**Comment by helmuth1792** [comment]
can you please tell me what bid options do you use?

**Comment by helmuth1792** [comment]
why does it say "Bid error: you are not allowed to bid on this trade"?
